pkg/logs/config/channel_message.go

```diff
@@ -26,7 +26,7 @@ type Lambda struct {
 }
 
 // NewChannelMessageFromLambda construts a message with content and with the given timestamp and Lambda metadata
-func NewChannelMessageFromLambda(content []byte, utcTime time.Time, ARN, reqID string) *ChannelMessage {
+func NewChannelMessageFromLambda(content []byte, utcTime time.Time, ARN string, reqID string) *ChannelMessage {
```

Comment:
nit: It's pretty common in go to omit arguments types when they are repeated for the sake of brevity. This is a pretty minor stylistic choice though.